test/pipelines-it-pai.yml

```diff
@@ -18,31 +34,31 @@ jobs:
     condition: eq( variables['build_docker_img'], 'true' )
     displayName: 'build nni bdsit_wheel'
 
-  - script: |
-      cd deployment/pypi
-      docker login -u $(docker_hub_user) -p $(docker_hub_pwd)
-
-      echo 'updating docker file for installing nni from local...'
-      # update Dockerfile to install NNI in docker image from whl file built in last step
-      sed -ie 's/RUN python3 -m pip --no-cache-dir install nni/COPY .\/dist\/* .\nRUN python3 -m pip install nni-*.whl/' ../docker/Dockerfile
-      cat ../docker/Dockerfile
-      echo $IMG_TAG
-      docker build -f ../docker/Dockerfile -t $(new_docker_img) .
-      docker push $(new_docker_img)
-    condition: eq( variables['build_docker_img'], 'true' )
-    displayName: 'build and upload nni docker image'
-
   - script: |
       source install.sh
     displayName: 'Install nni toolkit via source code'
 
   - script: |
       if [ $(build_docker_img) = 'true' ]
       then
-        export TEST_IMG=$(new_docker_img)
+        cd deployment/pypi
+        docker login -u $(docker_hub_user) -p $(docker_hub_pwd)
+        echo 'updating docker file for installing nni from local...'
+        # update Dockerfile to install NNI in docker image from whl file built in last step
+        sed -ie 's/RUN python3 -m pip --no-cache-dir install nni/COPY .\/dist\/* .\nRUN python3 -m pip install nni-*.whl/' ../docker/Dockerfile
+        cat ../docker/Dockerfile
+        export IMG_TAG=`date +%y%m%d%H%M`
```

Comment:
suggest to change to local time rather UTC time.

Reply:
Just checked, it is local time of the azure machine it is running on.